### PR TITLE
Tweak indicator label size

### PR DESCRIPTION
**Pull request by notlee**
- It is now 12px for 18px copy, the comment lied

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-labels bundle size difference from 5.1.3 to 880eb841051f67ea8897f0db5e6b71580384322d
No significant bundle size differences found.

File: src/scss/_variables.scss [hunk 1]
```diff
@@ -57,7 +57,7 @@ $_o-labels-indicator-status: (
 /// The width/height of the circle used in the indicator label.
 /// @type Number
 /// @access private
-$_o-labels-indicator-circle-size-calc: #{(6/12)}em #{'+'} 4px; // 10px given 12px font size, 12px given 18px font size
+$_o-labels-indicator-circle-size-calc: #{(4/12)}em #{'+'} 6px; // 10px given 12px font size, 12px given 18px font size
```
Comment (chee) [inline src/scss/_variables.scss:60]: image

**Review by chee — APPROVED** [review]

File: src/scss/_variables.scss (hunk 1 above)
Comment (notlee) [reply src/scss/_variables.scss:60]: 😬

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v5.1.4 :tada:

**Pull request merged**